.gitignore

```diff
@@ -1,4 +1,7 @@
 # Ignore .env files
 .env
 knowledge_packs/
-Manifest.toml
\ No newline at end of file
+Manifest.toml
+/Manifest.toml
+/docs/Manifest.toml
+/docs/build/
```

Comment:
Make sure to add `.vscode/**` and remove it from the index. It doesn't belong to the package!

Also, please add `**/.DS_Store` (it's a mac thing that keeps getting created)